app/controllers/admin/confirms_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Admin
+  class ConfirmsController < BaseController
+    before_action :set_account
+
+    def create
+      @account.user.update(confirmed_at: Time.now.utc)
```

Comment:
I believe devise provides a convenience method like `user.confirm` which does essentially what you are doing, with slightly more logic around it; so I might use that.